lib/database/repositories/Repository.js

```diff
@@ -109,6 +109,25 @@ class Repository {
         return this.model.create(entity);
     }
 
+    /**
+     * Removes a specific entities.
+     *
+     * @param {Object|QueryBuilder} [findQuery] the find query (see sequelize findAll options) or a find query builder
+     * @returns {Promise<*[]>} Promise with list of removed entities
+     */
```

Comment:
This function is critical enough to have a better documentation...

Reply:
refactored